<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddEditPressure</class>
 <widget class="QDialog" name="AddEditPressure">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>389</width>
    <height>116</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Систолическое</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QTimeEdit" name="timeCtrl"/>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Диастолическое</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Время</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelBtn">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="okBtn">
       <property name="text">
        <string>Ок</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="1">
    <widget class="QSpinBox" name="sisP">
     <property name="suffix">
      <string> мм рт. ст.</string>
     </property>
     <property name="maximum">
      <number>250</number>
     </property>
     <property name="singleStep">
      <number>5</number>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QSpinBox" name="diasP">
     <property name="suffix">
      <string> мм рт. ст.</string>
     </property>
     <property name="maximum">
      <number>250</number>
     </property>
     <property name="singleStep">
      <number>5</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
